runtime/tests/Test_LightningCoreQIS.cpp

```diff
@@ -522,7 +522,7 @@ TEST_CASE("Test __catalyst__qis__Measure", "[CoreQIS]")
 
         __catalyst__qis__PauliX(wire0, false);
 
-        Result m = __catalyst__qis__Measure(wire0);
+        Result m = __catalyst__qis__Measure(wire0, -1);
```

Comment:
could you add C++ tests for LightningQubit to check the `postselect` support?

Reply:
WIP :)

Reply:
Done.